pydantic/types.py

```diff
@@ -128,6 +129,19 @@ def conint(
     le: int | None = None,
     multiple_of: int | None = None,
 ) -> type[int]:
+    """A wrapper around `int` that allows for additional constraints.
+
+    Args:
+        strict: Whether to validate the integer in strict mode.
```

Comment:
Shouldn't we add the default value here?
```suggestion
        strict: Whether to validate the integer in strict mode. Defaults to `None`.
```
Same for other docstrings